### PR TITLE
[JENKINS-38632] Solve doclint errors

### PR DESCRIPTION
**Pull request by raul-arabaolaza**
[JENKINS-38632](https://issues.jenkins-ci.org/browse/JENKINS-38632)

Some warnings also solved. But others I am not able to fix because they imply writing javadoc and I do not have enough knowledge of this plugin

@reviewbybees specially @svanoort

### CODE REVIEW

**Comment by svanoort** [comment]
@raul-arabaolaza You don't actually need to address the lint complaints (warnings) which don't fail the build -- those should be all the ones that require knowledge of the code.  Waiting on the CI build to some extent while reviewing.

File: src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTiming.java [hunk 1]
```diff
@@ -62,26 +62,26 @@
 /**
  * Provides common, comprehensive set of APIs for doing status and timing computations on pieces of a pipeline execution.
  *
- * <p/> <strong>Concepts:</strong> a chunk, which is a set of {@link FlowNode}s in the same {@link FlowExecution} with a first and last node.
- * <p/> Chunks exist in a context: the FlowNode before and the FlowNode after.  These follow common-sense rules:
+ * <p> <strong>Concepts:</strong> a chunk, which is a set of {@link FlowNode}s in the same {@link FlowExecution} with a first and last node. </p>
```
Comment (svanoort) [inline src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTiming.java]: Prefer just a naked <p> tag.

**Comment by svanoort** [comment]
🐝  thanks for the PR - it looks good to me.   Looking at this when not rushing to ship block-scoped-stage, I see formatting mistakes fixed that didn't realize I had. 

Built locally with doclint on (since this plugin's default CI builder is java7 at the moment).

**Comment by svanoort** [comment]
Since it's just docs fixes... pulling author's prerogative here and merging after review.

**Comment by recena** [comment]
Belated 🐝  and 👍

**Pull request merged**